joomla/joomla-cms
# [4.3] Normalize styling of inputs

### Summary of Changes
Minor rework of input styling across both backend and frontend templates. The aim of this is to use more of the native bootstrap variables so that our rendering is more consistent. This also makes it easier for people to reuse our SCSS files that we distribute by using more of the bootstrap mixins (e.g. using the box-shadow and border-radius mixins).

### Testing Instructions
Go through native select, input, choices (both multiselect and single select - e.g. category and tags form fields in frontend editing) form fields in both cassieopia and atum. The changes should not be noticeable before and after.

For bonus points:
- Only find an extension still using chosen and test that too.
- Set `enable-rounded` variable to false in either casseopia or atum. Before patch only native BS fields were square and the rest were round. After they all act the same way.

### Link to documentations
Please select:
